src/sentry/api/endpoints/relay_projectconfigs.py

```diff
@@ -89,28 +86,16 @@ def _post(self, request):
             project.organization = organization
             project._organization_cache = organization
 
-            org_opts = org_options.get(organization.id) or {}
-
             with Hub.current.start_span(op="get_config"):
                 with metrics.timer("relay_project_configs.get_config.duration"):
                     project_config = config.get_project_config(
                         project,
-                        org_options=org_opts,
+                        org_options=org_options.get(organization.id) or {},
                         full_config=full_config_requested,
-                        project_keys=project_keys.get(project.id, []),
+                        project_keys=project_keys.get(project.id) or [],
                     )
 
-            configs[six.text_type(project_id)] = serialized_config = project_config.to_dict()
-
-            config_size = len(json.dumps(serialized_config))
-            metrics.timing("relay_project_configs.config_size", config_size)
-
-            # Log if we see huge project configs
-            if config_size >= PROJECT_CONFIG_SIZE_THRESHOLD:
-                logger.info(
```

Comment:
Do we still need this?

Reply:
We added this for debugging specific scenarios that no longer apply, so I guess we can remove it and simplify the endpoint.